scss/_main-courses.scss

```diff
@@ -0,0 +1,170 @@
+.main-courses {
+  width: 100%;
+  margin-top: 50px;
+
+  .card-img-top {
+    height: 250px;
+    border-top-left-radius: 15px;
+    border-top-right-radius: 15px;
+    border: none;
+
+    img {
+      width: 60%;
+    }
+  }
+
+  .html-css-js {
+    background-color: $html-css-js_color;
+  }
+
+  .lavarel {
+    background-color: $img-laravel_color;
+  }
+
+  .img-php {
+    background-color: $img-php_color;
+  }
+
+  .card-title {
+    font-size: 30px;
+    font-family: $font_UTM_BryantLG;
+    color: $card-title_color;
+    text-align: center;
+    font-weight: 700;
+  }
+
+  .card-text {
+    font-size: 14px;
+    font-family: $font_UTM_BryantLG;
+    color: $content_hapolearn;
+    text-align: center;
+  }
+
+  .card-text-overflow {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    -webkit-line-clamp: 2;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+  }
+
+  .btn-card-primary {
+    color: $white_color;
+    font-size: 18px;
+    font-weight: bold;
+    font-family: $font_UTM_BryantLG;
+    border-radius: 22px;
+    background-color: $background-border-btn-card;
+    margin-left: 20%;
+  }
+
+  .other-courses {
+    font-size: 60px;
+    font-family: $font_UTM_BryantLG;
+    font-weight: 700;
+    color: $content_hapolearn_title;
+    text-align: center;
+    -webkit-text-stroke: 2px;
+    padding-bottom: 15px;
+
+    .courses {
+      border-bottom: 3px solid $card-title_color;
+      border-bottom-left-radius: 3px;
+      border-bottom-right-radius: 3px;
+    }
+  }
+
+  .courses-boder {
+    .css-tutorial {
+      background-color: $background-css-tutorial;
+    }
+
+    .rails-courses {
+      background-color: $background-rails-tutorial;
+    }
+
+    .java-courses {
+      background-color: $background-java-tutorial;
+    }
+  }
+
+  .extend-courses {
+    font-size: 40px;
+    color: $text-expend-courses-color;
+    font-family: $font_UTM_BryantLG;
+    font-weight: bold;
+    line-height: 49.29px;
+    margin-top: 3%;
+    -webkit-text-stroke: 2px;
+
+    .right-arrow {
+      width: 47px;
+      height: 26px;
+    }
+  }
+}
+
+@media (max-width: 769px) and (min-width: 576px) {
+  .card-img-top {
+    height: 131px !important;
```

Comment:
sao cần !important như thế này vậy?

Reply:
không để importain thì chiều rộng của ảnh nó lấy chiều rộng ảnh của meadia 1440px

Reply:
@vupham3001 anh thấy ở các màn khác có dùng important cho `.card-img-top` này đâu nhỉ?

Reply:
em cũng không hiểu vì sao, lúc repon màn 769px nó toàn lấy chiều rộng của màn 1440px, em dùng importain thì mới được

Reply:
@vupham3001 do em nhầm level của nó đấy
ví dụ như các màn hình khác em dùng style như thế này
```
.main-courses {
  width: 100%;
  margin-top: 50px;

  .card-img-top { ... }
```
thì khi dịch ra nó sẽ ra kiểu css như sau
```
.main-courses .card-img-top { ... } 
```
còn ở đây thì em dùng  có mỗi
```
.card-img-top { ... } 

```

-> nó không đúng level của class `.card-img-top`